resource/composed/composed_test.go

```diff
@@ -104,3 +109,87 @@ func ExampleFrom() {
 	//   forProvider:
 	//     region: us-east-2
 }
+
+func TestFrom(t *testing.T) {
+	v1beta1.AddToScheme(Scheme)
+
+	type args struct {
+		o runtime.Object
+	}
+	type want struct {
+		cd  *Unstructured
+		err error
+	}
+	cases := map[string]struct {
+		reason string
+		args   args
+		want   want
+	}{
+		"WithMetadata": {
```

Comment:
do you want to add a test case that explicitly verifies the `generation` field is removed from metadata?

Reply:
That's effectively what this test case is doing, despite the fact I don't set `Generation` explicitly here. It would be getting set implicitly without #70. 😄